run_dir/design/flowcell.html

```diff
@@ -2,7 +2,7 @@
 {% block stuff %}
 {% import re %}
 <div id="page_content">
-    <h1>Flow Cell <span id="page_title">{{ flowcell_id }}</span></h1>
+    <h1>Flow Cell <span id="page_title">{{ flowcell.get('name') }}</span></h1>
```

Comment:
Is it worth using `flowcell.get('name', flowcell_id)` to default to `flowcell_id`?